coremltools/converters/mil/frontend/tensorflow/test/test_ops.py

```diff
@@ -2224,6 +2224,10 @@ class TestEinsum(TensorFlowBaseTest):
                 "abcd,abde->abce",
                 "abcd,efbd->eafc",
                 "acdb,bade->abce",
+                # Generic with diagonal
+                "jiii,ijjk->jk",
```

Comment:
Please add some unittests so we also cover the case where there are multiple diagonal dimensions in a single input,
for instance `ijij,iij->ij`